2022-02-25 15:43:13,335 - Bot Started As DakuIkzuku_bot

2022-02-25 15:44:01,762 - Starting new HTTPS connection (1): open.spotify.com:443
2022-02-25 15:44:02,939 - https://open.spotify.com:443 "GET /episode/6Dr4EYisCxpzhnxToq67EN HTTP/1.1" 200 None
2022-02-25 15:44:04,178 - 'NoneType' object is not subscriptable
2022-02-25 15:52:02,524 - Stop signal received (SIGINT). Exiting...
2022-02-25 15:52:03,162 - Bot Stopped, Bye.
2022-02-25 15:52:14,731 - Bot Started As DakuIkzuku_bot

2022-02-25 15:53:53,267 - Stop signal received (SIGINT). Exiting...
2022-02-25 15:53:53,894 - Bot Stopped, Bye.
2022-02-25 15:54:03,956 - Bot Started As DakuIkzuku_bot

2022-02-25 15:55:47,101 - Stop signal received (SIGINT). Exiting...
2022-02-25 15:55:47,126 - Bot Stopped, Bye.
2022-02-25 15:55:59,226 - Bot Started As DakuIkzuku_bot

2022-02-25 15:56:59,231 - Bot Started As DakuIkzuku_bot

2022-02-25 15:57:02,250 - 'list' object has no attribute 'write'
Traceback (most recent call last):
  File "/home/daniel/.local/lib/python3.8/site-packages/pyrogram/dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/daniel/Programs/Bots/MusicDownloader/mbot/plugins/greetings.py", line 95, in helpbtn
    await query.message.edit(text = text,reply_markup=button)
  File "/home/daniel/.local/lib/python3.8/site-packages/pyrogram/types/messages_and_media/message.py", line 2723, in edit_text
    return await self._client.edit_message_text(
  File "/home/daniel/.local/lib/python3.8/site-packages/pyrogram/methods/messages/edit_message_text.py", line 88, in edit_message_text
    reply_markup=await reply_markup.write(self) if reply_markup else None,
AttributeError: 'list' object has no attribute 'write'
2022-02-25 15:57:05,925 - 'list' object has no attribute 'write'
Traceback (most recent call last):
  File "/home/daniel/.local/lib/python3.8/site-packages/pyrogram/dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/daniel/Programs/Bots/MusicDownloader/mbot/plugins/greetings.py", line 95, in helpbtn
    await query.message.edit(text = text,reply_markup=button)
  File "/home/daniel/.local/lib/python3.8/site-packages/pyrogram/types/messages_and_media/message.py", line 2723, in edit_text
    return await self._client.edit_message_text(
  File "/home/daniel/.local/lib/python3.8/site-packages/pyrogram/methods/messages/edit_message_text.py", line 88, in edit_message_text
    reply_markup=await reply_markup.write(self) if reply_markup else None,
AttributeError: 'list' object has no attribute 'write'
2022-02-25 15:57:08,012 - 'list' object has no attribute 'write'
Traceback (most recent call last):
  File "/home/daniel/.local/lib/python3.8/site-packages/pyrogram/dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/daniel/Programs/Bots/MusicDownloader/mbot/plugins/greetings.py", line 95, in helpbtn
    await query.message.edit(text = text,reply_markup=button)
  File "/home/daniel/.local/lib/python3.8/site-packages/pyrogram/types/messages_and_media/message.py", line 2723, in edit_text
    return await self._client.edit_message_text(
  File "/home/daniel/.local/lib/python3.8/site-packages/pyrogram/methods/messages/edit_message_text.py", line 88, in edit_message_text
    reply_markup=await reply_markup.write(self) if reply_markup else None,
AttributeError: 'list' object has no attribute 'write'
2022-02-25 15:57:55,834 - Stop signal received (SIGINT). Exiting...
2022-02-25 15:57:56,329 - Bot Stopped, Bye.
2022-02-25 15:58:10,795 - Unable to connect due to network issues: timed out
2022-02-25 15:58:16,716 - Bot Started As DakuIkzuku_bot

2022-02-25 16:05:42,298 - Stop signal received (SIGINT). Exiting...
2022-02-25 16:05:42,415 - Bot Stopped, Bye.
2022-02-25 16:05:52,706 - Bot Started As DakuIkzuku_bot

2022-02-25 16:07:01,101 - 'InlineKeyboardMarkup' object has no attribute 'append'
Traceback (most recent call last):
  File "/home/daniel/.local/lib/python3.8/site-packages/pyrogram/dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/daniel/Programs/Bots/MusicDownloader/mbot/plugins/greetings.py", line 49, in start
    reply_markup.append([InlineKeyboardButton("LOG Channel", url=invite_link.invite_link)])
AttributeError: 'InlineKeyboardMarkup' object has no attribute 'append'
2022-02-25 16:07:45,202 - Stop signal received (SIGINT). Exiting...
2022-02-25 16:07:45,897 - Bot Stopped, Bye.
2022-02-25 16:07:54,542 - Bot Started As DakuIkzuku_bot

2022-02-25 16:09:41,506 - Starting new HTTPS connection (1): img.youtube.com:443
2022-02-25 16:09:42,161 - https://img.youtube.com:443 "GET /vi/R1pA9bt2_30/default.jpg HTTP/1.1" 200 2455
2022-02-25 16:09:49,694 - [0;31mERROR:[0m [youtube:truncated_id] R1pA9bt2_3: Incomplete YouTube ID R1pA9bt2_3. URL https://www.youtube.com/watch?v=R1pA9bt2_3 looks truncated.
Traceback (most recent call last):
  File "/home/daniel/.local/lib/python3.8/site-packages/yt_dlp/extractor/common.py", line 612, in extract
    ie_result = self._real_extract(url)
  File "/home/daniel/.local/lib/python3.8/site-packages/yt_dlp/extractor/youtube.py", line 5564, in _real_extract
    raise ExtractorError(
yt_dlp.utils.ExtractorError: Incomplete YouTube ID R1pA9bt2_3. URL https://www.youtube.com/watch?v=R1pA9bt2_3 looks truncated.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/daniel/.local/lib/python3.8/site-packages/yt_dlp/YoutubeDL.py", line 1381, in wrapper
    return func(self, *args, **kwargs)
  File "/home/daniel/.local/lib/python3.8/site-packages/yt_dlp/YoutubeDL.py", line 1451, in __extract_info
    ie_result = ie.extract(url)
  File "/home/daniel/.local/lib/python3.8/site-packages/yt_dlp/extractor/common.py", line 638, in extract
    raise type(e)(e.msg, **kwargs)
yt_dlp.utils.ExtractorError: [youtube:truncated_id] R1pA9bt2_3: Incomplete YouTube ID R1pA9bt2_3. URL https://www.youtube.com/watch?v=R1pA9bt2_3 looks truncated.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/daniel/.local/lib/python3.8/site-packages/pyrogram/dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/daniel/Programs/Bots/MusicDownloader/mbot/plugins/youtube.py", line 43, in _
    ids = await getIds(message.matches[0].group(0))
  File "/home/daniel/.local/lib/python3.8/site-packages/asgiref/sync.py", line 414, in __call__
    ret = await asyncio.wait_for(future, timeout=None)
  File "/usr/lib/python3.8/asyncio/tasks.py", line 455, in wait_for
    return await fut
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/daniel/.local/lib/python3.8/site-packages/asgiref/sync.py", line 455, in thread_handler
    return func(*args, **kwargs)
  File "/home/daniel/Programs/Bots/MusicDownloader/mbot/utils/ytdl.py", line 31, in getIds
    info_dict = ydl.extract_info(video, download=False)
  File "/home/daniel/.local/lib/python3.8/site-packages/yt_dlp/YoutubeDL.py", line 1372, in extract_info
    return self.__extract_info(url, self.get_info_extractor(ie_key), download, extra_info, process)
  File "/home/daniel/.local/lib/python3.8/site-packages/yt_dlp/YoutubeDL.py", line 1399, in wrapper
    self.report_error(str(e), e.format_traceback())
  File "/home/daniel/.local/lib/python3.8/site-packages/yt_dlp/YoutubeDL.py", line 930, in report_error
    self.trouble(f'{self._format_err("ERROR:", self.Styles.ERROR)} {message}', *args, **kwargs)
  File "/home/daniel/.local/lib/python3.8/site-packages/yt_dlp/YoutubeDL.py", line 871, in trouble
    raise DownloadError(message, exc_info)
yt_dlp.utils.DownloadError: [0;31mERROR:[0m [youtube:truncated_id] R1pA9bt2_3: Incomplete YouTube ID R1pA9bt2_3. URL https://www.youtube.com/watch?v=R1pA9bt2_3 looks truncated.
2022-02-25 16:11:26,552 - Bot Started As DakuIkzuku_bot

2022-02-25 16:11:38,785 - [0;31mERROR:[0m [youtube:truncated_id] R1pA9bt2_: Incomplete YouTube ID R1pA9bt2_. URL https://www.youtube.com/watch?v=R1pA9bt2_ looks truncated.
2022-02-25 16:11:46,632 - Stop signal received (SIGINT). Exiting...
2022-02-25 16:11:47,606 - Bot Stopped, Bye.
2022-02-25 16:11:56,780 - Bot Started As DakuIkzuku_bot

2022-02-25 16:11:59,883 - [0;31mERROR:[0m [youtube:truncated_id] R1pA9bt2_: Incomplete YouTube ID R1pA9bt2_. URL https://www.youtube.com/watch?v=R1pA9bt2_ looks truncated.
2022-02-25 16:18:56,253 - Stop signal received (SIGINT). Exiting...
2022-02-25 16:18:56,390 - Bot Stopped, Bye.
